test/conftest.py

```diff
@@ -113,7 +113,7 @@ def pytest_collection_modifyitems(config, items):
     },
     OfflineExperienceReplayLearner: {
         "memory_size": 30,
-        "memory_batch_size": 20,
+        "batch_memory_frac": 0.6,
```

Comment:
What is the `batch_size` here? if the total batch size is 30 and the memory batch size is 20 this implies `batch_memory_frac = 0.666666`. But the values here would imply `batch_size = 33.33333`. (Or is it me doing the computation wrong here?). Thanks

Reply:
true, this should imply that memory_batch_size will be 18

Reply:
I didn't pay close attention to the numbers, since the tests don't depend on exact memory sizes.